FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -5,6 +5,34 @@
 import Foundation
 
 public final class RemoteFeedLoader: FeedLoader {
+	
+	private struct Root: Codable {
```

Comment:
`Codable` is a type alias for the `Encodable` and `Decodable` protocols. In this case, we only need to decode `Root` from an external representation so `Codable` can be replaced with `Decodable`:

```suggestion
	private struct Root: Decodable {
```